cmd/ratify/cmd/serve.go

```diff
@@ -103,7 +103,12 @@ func serve(opts serveCmdOptions) error {
 	}
 
 	if opts.httpServerAddress != "" {
-		server, err := httpserver.NewServer(context.Background(), opts.httpServerAddress, getExecutor, opts.certDirectory, opts.caCertFile, opts.cacheTTL, opts.metricsEnabled, opts.metricsType, opts.metricsPort)
+		logConfig, err := config.GetLoggerConfig(opts.configFilePath)
+		if err != nil {
+			return err
```

Comment:
should we add some additional msg on error return?

Reply:
yeah, I'll add it.